### DISCUSSION TITLE
Random note from smart connections

### DISCUSSION THREAD

**Discussion by jakemkc** [brianpetro/obsidian-smart-connections#205] [General]
Hi,

I use the Obsidian built-in plugin to open random note. Today I discovered that there is a command "smart connects: random note from smart connections"

I can't find description of this command. What are the differences between this one from smart connections and the built-in? 

Best,
Jake

**Comment by brianpetro** [comment]
Good noticing! It selects a random note based on the Smart Connections of current file.

**Comment by jiwonac** [comment]
Question: Is the random sampling uniform over top-$k$ most similar notes, or weighted based on similarity?

**Reply by brianpetro** [reply]
Random over Top-K.

🌴
